dev/setup-k3ds

```diff
@@ -22,7 +22,7 @@ fi
 # https://hub.docker.com/r/rancher/k3s/tags
 #args="$args -i docker.io/rancher/k3s:v1.22.15-k3s1"
 
-k3d cluster create upstream --servers 3 --api-port 36443 -p '80:80@server:0' -p '443:443@server:0' $args
+k3d cluster create upstream --servers 1 --api-port 36443 -p '80:80@server:0' -p '443:443@server:0' $args
```

Comment:
It is possible to set up a 1-node k3s/k3d cluster backed by etcd by adding the `--cluster-init@server:0` argument if desired.

Reply:
Interesting info about embedded etc vs sqlite.

It's already set to `1` in `dev/setup-k3ds`, but it doesn't have to be the same.
I put it to 1, because images have to be uploaded into each node. Especially for scripts like https://github.com/rancher/fleet/blob/master/dev/update-agent-k3d one node is noticeable faster.